api/__tests__/fixtures/select-project-set-quota-sizes.json

```diff
@@ -1,28 +1,28 @@
 [
     {
         "dev": {
-            "quotaCpuSize": "small",
-            "quotaMemorySize": "small",
-            "quotaStorageSize": "small",
-            "quotaSnapshotSize": "small"
+            "quotaCpuSize": "cpu-request-0.5-limit-1.5",
+            "quotaMemorySize": "memory-request-2-limit-4",
```

Comment:
is there something special about the terms `cpu-request-0.5-limit-1.5`? is it parsed? if it is. me no like

Reply:
This is our default small size for CPU, and this file is fixtures where I use it for unit test